lstchain/reco/r0_to_dl1.py

```diff
@@ -618,28 +619,50 @@ def r0_to_dl1(
+                                         use_main_island=True)
+                except HillasParameterizationError:
+                    logging.exception(
+                        'HillasParameterizationError in get_dl1()'
+                    )
+
                     image = event.dl1.tel[telescope_id].image
 
-                    if ('lh_fit_config' in config.keys()
-                       and dl1_filled['n_pixels'] is not 0
-                       and dl1_filled['n_pixels'] < 1000):
-                        is_saturated = np.any(image > config['lh_fit_config']['n_peaks'])
-
-                        if not is_saturated: #reject computationnally expensive events which would be poorly estimate with the selected value of n_peak TODO : improve to not reject events
-
-                            dl1_filled = get_dl1_lh_fit(event,
-                                                        subarray,
-                                                        telescope_id,
-                                                        image=image,
-                                                        is_simu=is_simu,
-                                                        normalized_pulse_template=pulse_template,
-                                                        dl1_container=dl1_filled,
-                                                        custom_config=config,
-                                                        use_main_island=True)
+                    if 'lh_fit_config' in config.keys():
+                        if (dl1_filled['n_pixels'] is not 0
+                            and dl1_filled['n_pixels'] < 1000):
+
+                            is_saturated = np.any(image > config['lh_fit_config']['n_peaks'])
+
+                            if not is_saturated:
+                                # rejects computationnally expensive events which would
+                                # be poorly estimated with the selected value of n_peak
+                                # TODO : improve to not reject events
+                                try:
+                                    dl1_filled = get_dl1_lh_fit(event,
+                                                                subarray,
+                                                                telescope_id,
+                                                                image=image,
+                                                                is_simu=is_simu,
+                                                                normalized_pulse_template=pulse_template,
+                                                                dl1_container=dl1_filled,
+                                                                custom_config=config,
+                                                                use_main_island=True)
+                                    dl1_filled.lhfit_call_status = "Processed"
+                                except HillasParameterizationError:
```

Comment:
Please check the code and see which kind of errors in can throw. There'll be probably `HillasParametrizationError` from the regular `get_dl1` call plus possible errors from LH-fit code itself